easylist/easylist_specific_block.txt

```diff
@@ -5,6 +5,7 @@
 ||1337x.*/images/x2x8.jpg
 ||2merkato.com/images/banners/
 ||2oceansvibe.com/?custom=takeover
+||373news.com/mns_images/*
```

Comment:
Badly braeks the site, please remove.

Reply:
The filter was working fine for me. I cant see any other breakage.
I removed as you said it's self-promotion.